RELEASE.md

```diff
@@ -56,7 +56,7 @@ Create release in [GitHub](https://github.com/habitat-sh/habitat/releases)
 On the Github releases page, there should already be a tag for the release (pushed up previously).
 Draft a new Release, specify the tag, and title it with the same (eg, 0.18.0). Then hit Publish Release.
 
-# Drink beer
+# Drink a fine glass of Cabernet Sauvignon, preferably from the Stag's Leap District
```

Comment:
We may have to host a Habitat Summit to discuss this. I suggest a cozy lodge somewhere in the Rocky Mountains with a roaring fire and a view of a lake.